### PR TITLE
fix(staking): prevent anchor save in browser history

### LINKED ISSUE #60: backspace returns the window to the delegation
When I went to the page to the validator and pressed the delegate button (but changed my mind), and decided to exit the validator and look for another, then at the moment the back key was pressed (in the browser), the delegation window opened, instead of returning to the page with the list of validators

image
image

### PR DESCRIPTION
**Pull request by hadzhehsen**
closes: #60

### CODE REVIEW

File: libs/staking/validator-details-page/src/components/validator-info/validator-info.tsx [hunk 1]
```diff
@@ -585,7 +592,6 @@ export function ValidatorInfo({
     </Fragment>
   );
 }
-
```
Comment (olegshilov) [inline libs/staking/validator-details-page/src/components/validator-info/validator-info.tsx]: revert

File: libs/staking/validator-details-page/src/components/validator-info/validator-info.tsx [hunk 2]
```diff
@@ -1,6 +1,13 @@
...
 import { useAccount, useBalance, useNetwork } from 'wagmi';
-import { Link, useLocation, useNavigate } from 'react-router-dom';
+import {
+  Link,
+  NavigateFunction,
+  RelativeRoutingType,
+  To,
+  useLocation,
+  useNavigate,
+} from 'react-router-dom';
```
Comment (olegshilov) [inline libs/staking/validator-details-page/src/components/validator-info/validator-info.tsx]: revert

**Review by olegshilov — CHANGES_REQUESTED** [review]

File: libs/staking/validator-details-page/src/components/validator-info/validator-info.tsx (hunk 2 above)
Comment (hadzhehsen) [reply libs/staking/validator-details-page/src/components/validator-info/validator-info.tsx]: Done

File: libs/staking/validator-details-page/src/components/validator-info/validator-info.tsx (hunk 1 above)
Comment (hadzhehsen) [reply libs/staking/validator-details-page/src/components/validator-info/validator-info.tsx]: Done

**Review by olegshilov — DISMISSED** [review]

**Review by olegshilov — APPROVED** [review]

**Pull request merged**